### DISCUSSION TITLE
IndexError:Index out of range while annotating the Video(Files size is 11.7 Mb)

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4292] [Q&A]
While doing the video annotation, we are using both shape and track method for annotations.

After annotating more than 100 frames, We are getting error while task dataset that is by clicking 'export_task_dataset' .and the error is "IndexError:Index out of range".

Could you please tell us the reason?

Thank you.

**Comment by nmanovic** [comment]
@keshva23 , please create an issue and help us to reproduce on our end.
